### PR TITLE
Spark side work to use new parquet spark dumps

### PR DESCRIPTION
**Pull request by amCap1712**
Spark side work to utilise new parquet dumps. This should be merged together with #1545.

### CODE REVIEW

**Comment by amCap1712** [comment]
Till now this PR contains the logic to upload parquet dumps to Spark and load the listens. Further work, includes rewriting stats/recs/similar users to use these dumps.

**Review by mayhem — APPROVED** [review]
Overall this looks sane and it appears that things are getting simpler, which is really really good to see!

File: listenbrainz_spark/stats/user/recording.py [hunk 1]
```diff
@@ -13,64 +12,56 @@ def get_recordings(table):
...
-                     , nullif(release_name, '') as release_name
-                     , CASE
-                         WHEN release_mbid IS NOT NULL AND release_mbid != '' THEN NULL
-                         ELSE nullif(release_msid, '')
-                       END as release_msid
-                     , nullif(release_mbid, '') as release_mbid
-                  FROM {}
-              )
-            SELECT *
+    result = run_query(f"""
```
Comment (mayhem) [inline listenbrainz_spark/stats/user/recording.py:29]: oh wow. I love how this PR simplifies things overall, but this is serious improvement!

**Review by mayhem — APPROVED** [review]
I love this PR sooo much! Looking great!

**Pull request merged**